robottelo/cleanup.py

```diff
@@ -58,21 +57,3 @@ def vm_cleanup(vm):
     :param robottelo.vm.VirtualMachine vm: virtual machine to destroy
     """
     vm.destroy()
-
-
-def cleanup_of_provisioned_server(hostname=None, provisioning_server=None, distro=None):
```

Comment:
Why are we removing this one from here ?

Reply:
I want to keep everything related to upgrade into upgrade_utility file.

Reply:
@devendra104 `cleanup_of_provisioned_server` is more a common utils and could be used by tiers tests as well .. hence would not suggests to move it from here ..

Reply:
Agree, But the current scope of this module was only upgrade tests, that was the only reason for keeping this method/function in upgrade_utility. 
Anyway, I rollback this change.